api/pkg/apis/v1alpha1/providers/target/rust/Cargo.toml

```diff
@@ -7,7 +7,8 @@ resolver = "2"
 members = [
     "symphony",
     "rust_providers/mock",
-    "rust_providers/ankaios"
+    "rust_providers/ankaios",
+    "rust_providers/uprotocol"
```

Comment:
Is it possible to have a self-contained uProtocol mock server for some unit testing?

Reply:
Yeah, similar to TestHttpTargetProviderApplyInvalidStatusCode in https://github.com/eclipse-symphony/symphony/blob/main/api/pkg/apis/v1alpha1/providers/target/http/http_test.go, where we have a mock HTTP server to generate some response codes.